File: 2193/2193.c
int minimumNumberOfMovesToMakePali(struct TreeNode* root) {
    if (!root) return 0;
    int l = minimumNumberOfMovesToMakePali(root->left);
    int r = minimumNumberOfMovesToMakePali(root->right);
    return 1 + (l > r ? l : r);
}

File: 2193/2193.cs
public class Solution {
    public int MinimumNumberOfMovesToMakePali(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumNumberOfMovesToMakePali(root.left), MinimumNumberOfMovesToMakePali(root.right));
    }
}